core/server/utils/startup-check.js

```diff
@@ -2,6 +2,8 @@ var packages = require('../../../package.json'),
     path = require('path'),
     crypto = require('crypto'),
     fs = require('fs'),
+    semver = require('semver'),
```

Comment:
Only packages that are built into node can be required here, otherwise the checks for missing dependencies won't be able to run.